docs/how_to_guides/versioned_idaes_install.rst

```diff
@@ -37,16 +37,16 @@ Installing IDAES
 ----------------
 .. _updating_install:
 
-* To get a previous `IDAES release <https://github.com/IDAES/idaes-pse/releases>`_, for example v1.13 ::
+* To get a previous `IDAES release <https://github.com/IDAES/idaes-pse/releases>`_, for example 2.0 ::
 
-   pip install idaes-pse==1.13
+   pip install idaes-pse==2.0
 
 * To get the latest version from the GitHub main branch ::
```

Comment:
One thing I do not see here is instructions on how to get the optional dependencies; this is one of our user facing sets of install instructions so we probably need to cover it here (or point to the README for more details and to avoid the possibility of conflicting instrucitons).

Reply:
I've added a How-to page on optional dependencies: https://idaes-pse--1133.org.readthedocs.build/en/1133/how_to_guides/opt_dependencies.html